force-app/main/default/classes/RelationshipSettingsService_TEST.cls

```diff
@@ -0,0 +1,140 @@
+     * RelationshipSettingsService class when one already exists.
+     ***************************************************************************************************************************/
+    @isTest
+    private static void getInstanceExisting() {
+        Test.startTest();
+        RelationshipSettingsService relationshipSettingsServiceInstance1 = RelationshipSettingsService.getInstance();
+        RelationshipSettingsService relationshipSettingsServiceInstance2 = RelationshipSettingsService.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(
+            RelationshipSettingsService.instance,
+            relationshipSettingsServiceInstance1,
+            'Instance of mapper class returned should match static instance variable.'
+        );
+        System.assertEquals(
+            relationshipSettingsServiceInstance1,
+            relationshipSettingsServiceInstance2,
+            'Subsequent retrievals of mapper class instance should return existing instance.'
+        );
+        System.assertEquals(
+            true,
+            relationshipSettingsServiceInstance1 != null,
+            'Instance of mapper class should not be null.'
+        );
+    }
+
+    /***********************************************************************************************************************************
+     * @description Test method to verify that the locateRelationshipSettingsMapper method returns instance of RelationshipSettingsMapper
+     ***********************************************************************************************************************************/
+    @isTest
+    private static void locateRelationshipSettingsMapper() {
+        RelationshipSettingsService RelationshipSettingsServiceInstance = RelationshipSettingsService.getInstance();
+
+        Test.startTest();
+        RelationshipSettingsMapper relationshipSettingsMapperInstance = RelationshipSettingsServiceInstance.locateRelationshipSettingsMapper();
+        Test.stopTest();
+
+        System.assertEquals(
+            RelationshipSettingsMapper.getInstance(),
+            relationshipSettingsMapperInstance,
+            'Should return the instance of RelationshipSettingsMapperInstance'
+        );
+    }
+
+    /***********************************************************************************************************************************
+     * @description Test method to verify that the getRelationshipSettingsModel method returns a valid RelationshipSettingsModel
+     ***********************************************************************************************************************************/
+    @isTest
+    private static void getRelationshipSettingsModelValid() {
+        RelationshipSettingsMapper.instance = new STUB_RelationshipSettingsMapper();
+
+        Hierarchy_Settings__c hierarchySettings = UTIL_CustomSettingsFacade.getSettingsForTests(
```

Comment:
We dont need this right? Since you are stubbing the mapper and forcing it to return a model you want.

Reply:
You're right! Removing it.